lib/connect.js

```diff
@@ -235,7 +234,6 @@ function connect(token, options) {
 
   options = Object.assign({
     automaticSubscription: true,
-    createLocalTracks,
```

Comment:
It's because we've moved the dependency into the `index.ts` file instead of having `createLocalTracks` here. This was the suggestion @makarandp0 made to get around relying on a ts module from within our js files.

Reply:
thats because `index.ts` now passes createLocalTracks to `connect`